modules/Bio/EnsEMBL/Variation/Pipeline/SpliceAI/SpliceAI_conf.pm

```diff
@@ -0,0 +1,195 @@
+        gene_annotation         => $self->o('gene_annotation'), # '/homes/dlemos/work/tools/SpliceAI_files_output/gene_annotation/ensembl_gene/grch38_MANE_8_7.txt'
+        step_size               => $self->o('step_size'), # number of variants used to split the main vcf files
+        check_transcripts       => $self->o('check_transcripts'), # checks which are the new MANE transcripts for the last months, runs SpliceAI only for these ones
+        registry                => $self->o('registry'), # database where new MANE transcripts are going to be checked
+        # transcripts_file        => $self->o('transcripts_file'), # Instead of checking the new transcripts from the database, it can run only for transcripts from this file
+        output_file_name        => 'spliceai_scores_',
+
+        pipeline_wide_analysis_capacity => 100,
+
+        pipeline_db => {
+            -host   => $self->o('hive_db_host'),
+            -port   => $self->o('hive_db_port'),
+            -user   => $self->o('hive_db_user'),
+            -pass   => $self->o('hive_db_password'),
+            -dbname => $ENV{'USER'} . '_hive_' . $self->o('pipeline_name'),
+            -driver => 'mysql',
+        },
+    };
+}
+
+sub resource_classes {
+    my ($self) = @_;
+    return {
+        %{$self->SUPER::resource_classes},
+        '4Gb_8c_job'  => {'LSF' => '-n 8 -q production-rh74 -R"select[mem>4000]  rusage[mem=4000]" -M4000' },
+        '4Gb_job'     => {'LSF' => '-q production-rh74 -R"select[mem>4000] rusage[mem=4000]" -M4000'},
+    };
+}
+
+sub pipeline_analyses {
+  my ($self) = @_;
+  my @analyses;
+  push @analyses, (
+      # pre run checks, directories exist etc
+
+      {   -logic_name => 'init_files',
+          -module     => 'Bio::EnsEMBL::Hive::RunnableDB::JobFactory',
+          -input_ids  => [{}],
+          -parameters => {
+            'input_directory' => $self->o('input_directory'),
+            'inputcmd'        => 'find #input_directory# -type f -name "all_snps_ensembl_38_*.vcf" -printf "%f\n"',
+          },
+          -flow_into  => {
+            '2->A' => {'split_files' => {'vcf_file' => '#_0#'}},
+            'A->1' => ['get_chr_dir'],
+          },
+      },
+      { -logic_name => 'split_files',
+        -module => 'Bio::EnsEMBL::Variation::Pipeline::SpliceAI::SplitFiles',
+        -input_ids  => [],
+        -parameters => {
+          'main_dir'          => $self->o('main_dir'),
+          'input_directory'   => $self->o('input_directory'),
+          'tmp_split_vcf_dir' => $self->o('tmp_split_vcf_dir'),
+          'split_vcf_dir'     => $self->o('tmp_split_vcf_input_dir'),
+          'output_dir'        => $self->o('tmp_output_dir'),
+          'step_size'         => $self->o('step_size'),
+          'check_transcripts' => $self->o('check_transcripts'),
+          # 'transcripts_file'  => $self->o('transcripts_file'),
```

Comment:
Needed?

Reply:
Delete. Maybe later I'll add the option to run SpliceAI for a list of transcripts.